Dockerfile

```diff
@@ -9,4 +9,9 @@ RUN sed -i -e 's!\bmain\b!main contrib!g' /etc/apt/sources.list && \
     && clean-install \
     xfsprogs \
     open-iscsi \
-    azure-cli \
+    azure-cli
+
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list &&\
+    apt-get update && \
+    apt-get -t stretch-backports install glusterfs-client glusterfs-common -y && \
```

Comment:
Is there any policy on backports that prevents breaking update if we don't pin this version?

Reply:
Not that I know of.